### PR TITLE
build docker image for CUDA11

### PR DESCRIPTION
**Pull request by zasdfgbnm**

### CODE REVIEW

**Review by malfet — APPROVED** [review]

File: .circleci/cimodel/data/simple/util/docker_constants.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 # ARE YOU EDITING THIS NUMBER?  MAKE SURE YOU READ THE GUIDANCE AT THE
 # TOP OF .circleci/config.yml
-DOCKER_IMAGE_TAG = "209062ef-ab58-422a-b295-36c4eed6e906"
+DOCKER_IMAGE_TAG = "fff7795428560442086f7b2bb6004b65245dc11a"
```
Comment (malfet) [inline .circleci/cimodel/data/simple/util/docker_constants.py:5]: Hmm, has image tag pattern changed recently?

File: .circleci/docker/build.sh [hunk 2]
```diff
@@ -277,6 +277,14 @@ docker build \
        "$@" \
        .
 
+# NVIDIA dockers for RC releases use tag names like `11.0-cudnn8-devel-ubuntu18.04-rc`,
+# for this case we will set UBUNTU_VERSION to `18.04-rc` so that the Dockerfile could
+# find the correct image. As a result, here we have to replace the
+#   "$UBUNTU_VERSION" == "18.04-rc"
+# with
+#   "$UBUNTU_VERSION" == "18.04"
+UBUNTU_VERSION=$(echo ${UBUNTU_VERSION} | grep -oP '\d*\.\d*$|\d*\.\d*(?=-rc)')
```
Comment (seemethere) [inline .circleci/docker/build.sh:286]: Would it be better here to just strip any instance of `-rc` here?

```suggestion
UBUNTU_VERSION=$(echo ${UBUNTU_VERSION} | sed 's/-rc$//')
```

File: .circleci/cimodel/data/simple/util/docker_constants.py (hunk 1 above)
Comment (seemethere) [reply .circleci/cimodel/data/simple/util/docker_constants.py:5]: This looks like the new docker image tags introduced in https://github.com/pytorch/pytorch/pull/40194

```
$ git rev-parse HEAD:.circleci/docker
fff7795428560442086f7b2bb6004b65245dc11a
```

These can be used now if wanted.

File: .circleci/docker/build.sh (hunk 2 above)
Comment (zasdfgbnm) [reply .circleci/docker/build.sh]: Applied and this PR should be ready now

**Review by seemethere — APPROVED** [review]

**Comment by zasdfgbnm** [comment]
@seemethere or @malfet Could any of you help me land this? I can not land PR by myself.

**Comment by malfet** [comment]
@zasdfgbnm can you please rebase your change (there are 6 failures detected by CircleCI and I'm a bit hesitant to merge it in this state)

**Comment by zasdfgbnm** [comment]
@malfet Sure, rebased

**Comment by zasdfgbnm** [comment]
CI looks much better now

**Pull request closed without merging**